MonoGame.Framework.Content.Pipeline/Processors/FontDescriptionProcessor.cs

```diff
@@ -39,11 +39,13 @@ public override SpriteFontContent Process(FontDescription input, ContentProcesso
                 var directories = new List<string> { Path.GetDirectoryName(input.Identity.SourceFilename) };
                 var extensions = new string[] { "", ".ttf", ".ttc", ".otf" };
 
-                // Add special per platform directories
-                if (CurrentPlatform.OS == OS.Windows)
-                    directories.Add(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "Fonts"));
-                else if (CurrentPlatform.OS == OS.MacOSX)
-                    directories.Add("/Library/Fonts");
+		// Add special per platform directories
```

Comment:
the indentation on that entire file is messed up. I think this is just a small part of the problem :/